### ISSUE TITLE
Agent: Add ranking

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ArctosDB/arctos#261]
```
Background: 
Transactions occasionally produce somewhat subjective information about the
agents involved. Examples include the condition of returning loaned
specimens, acknowledgment of loaned items in publications, and any other
information which Curators might wish to consider for future transactions.

Proposed Solution:
Allow curatorial users to rank agents on a 5-point scale and comment on the
rank given. Make this available from various Arctos forms, including Loans,
Citations, Borrows, and Agents.

Concerns:
Who do we allow to add rankings? 
Who, if anyone, do we allow to edit them?
Do we allow anonymous rankings?
Do we display rankings, comments, summary, whodunit, ???
Do we moderate the comments? If so, who moderates?

In short, how do we keep this restricted to honest evaluations of past
performance?
```

Original issue reported on code.google.com by `dust...@gmail.com` on 24 Apr 2009 at 4:09
